pkg/stores/mod/db.go

```diff
@@ -37,10 +37,10 @@ func (ns *NotorietyStore) AddNotoriousGame(playerID string, gameID string, gameT
 	return result.Error
 }
 
-func (ns *NotorietyStore) GetNotoriousGames(playerID string) ([]*ms.NotoriousGame, error) {
+func (ns *NotorietyStore) GetNotoriousGames(playerID string, limit int) ([]*ms.NotoriousGame, error) {
 	var games []notoriousgame
 
-	result := ns.db.Table("notoriousgames").
+	result := ns.db.Limit(limit).Table("notoriousgames").
```

Comment:
this should return last n games instead of first n games. (order desc, then `items[len(games) - 1 - idx] = &ms.NotoriousGame{...}`)

Reply:
Changes made (I hope).